src/components/collections-grid/shop-by-room.js

```diff
@@ -40,7 +40,7 @@ export const ShopByRoom = ({
       <div className={sty.buttonArea}>
         <div className={sty.horizontalLine}></div>
 
-        <PrismicLink className="BtnPrimary" href={btnLink}>
+        <PrismicLink className="BtnPrimary" href={btnLink?.url}>
```

Comment:
not related to the slice but noticed this button that takes you to catalog page wasn't working so fixed it here